core/src/main/java/org/apache/iceberg/actions/SortStrategy.java

```diff
@@ -108,22 +84,12 @@ public RewriteStrategy options(Map<String, String> options) {
 
   @Override
   public Iterable<FileScanTask> selectFilesToRewrite(Iterable<FileScanTask> dataFiles) {
-    if (rewriteAll) {
-      LOG.info("Sort Strategy for table {} set to rewrite all data files", table().name());
-      return dataFiles;
-    } else {
-      return super.selectFilesToRewrite(dataFiles);
-    }
+    return super.selectFilesToRewrite(dataFiles);
   }
 
   @Override
   public Iterable<List<FileScanTask>> planFileGroups(Iterable<FileScanTask> dataFiles) {
```

Comment:
This is also not needed if we are only calling the super method

Reply:
Yeah, not required really. I have removed it.